chain/jsonrpc/client/src/lib.rs

```diff
@@ -189,6 +189,8 @@ jsonrpc_client!(pub struct JsonRpcClient {
     pub fn chunk(&self, id: ChunkId) -> RpcRequest<ChunkView>;
     pub fn validators(&self, block_id: MaybeBlockId) -> RpcRequest<EpochValidatorInfo>;
     pub fn gas_price(&self, block_id: MaybeBlockId) -> RpcRequest<GasPriceView>;
+    #[allow(non_snake_case)]
+    pub fn EXPERIMENTAL_receipt(&self, id: near_jsonrpc_primitives::types::receipts::ReceiptReference) -> RpcRequest<near_primitives::views::ReceiptView>;
```

Comment:
This style is deprecated since it only generates legacy positional arguments (sorry for the missing comment). Please, add the `impl` below following the example of `fn query`, where you pass `RpcReceiptRequest` and return `RpcRequest<RpcReceiptResponse>`.